math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -680,7 +680,7 @@ RVec<typename RVec<T>::size_type> Argsort(const RVec<T> &v)
 
 /// Return elements of a vector at given indices
 template <typename T>
-RVec<T> ByIndices(const RVec<T> &v, const RVec<typename RVec<T>::size_type> &i)
+RVec<T> Take(const RVec<T> &v, const RVec<typename RVec<T>::size_type> &i)
```

Comment:
I think `Take` is too generic (i.e., same as `Get`), it's worse than `ByIndices`.

Reply:
so probably `TakeIndices`? Or `TakeByIndices`?